src/countries/dto/update-country.dto.ts

```diff
@@ -0,0 +1,4 @@
+export class UpdateCountryDto {
+  name: string;
+  logo: string;
+}
```

Comment:
This non-breaking issue, and others happening on my device (Windows), could be resolved by configuring eslint/prettier/tsconfig to auto-fix all of these for you.
I'll create a ticket and we can either do it on a meet or I can tell you what needs to be done in the ticket.